{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/39674","repository_url":"https://api.github.com/repos/elastic/elasticsearch","labels_url":"https://api.github.com/repos/elastic/elasticsearch/issues/39674/labels{/name}","comments_url":"https://api.github.com/repos/elastic/elasticsearch/issues/39674/comments","events_url":"https://api.github.com/repos/elastic/elasticsearch/issues/39674/events","html_url":"https://github.com/elastic/elasticsearch/issues/39674","id":417136134,"node_id":"MDU6SXNzdWU0MTcxMzYxMzQ=","number":39674,"title":"Encountered a retryable error. Will Retry with exponential backoff ；code=>400 vs 6.5.1","user":{"login":"MrSsunlight","id":32731294,"node_id":"MDQ6VXNlcjMyNzMxMjk0","avatar_url":"https://avatars2.githubusercontent.com/u/32731294?v=4","gravatar_id":"","url":"https://api.github.com/users/MrSsunlight","html_url":"https://github.com/MrSsunlight","followers_url":"https://api.github.com/users/MrSsunlight/followers","following_url":"https://api.github.com/users/MrSsunlight/following{/other_user}","gists_url":"https://api.github.com/users/MrSsunlight/gists{/gist_id}","starred_url":"https://api.github.com/users/MrSsunlight/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/MrSsunlight/subscriptions","organizations_url":"https://api.github.com/users/MrSsunlight/orgs","repos_url":"https://api.github.com/users/MrSsunlight/repos","events_url":"https://api.github.com/users/MrSsunlight/events{/privacy}","received_events_url":"https://api.github.com/users/MrSsunlight/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2019-03-05T06:19:19Z","updated_at":"2019-03-05T12:35:44Z","closed_at":"2019-03-05T12:35:44Z","author_association":"NONE","active_lock_reason":null,"body":"version:  6.5.1\r\nPlugins :  logstash-output-elasticsearch\r\nOS version: centos 7.0\r\njava -version: 1.8.0\r\n\r\n\r\nI am using Logstash to transferring data to Elasticsearch.\r\n\r\nlogstash.conf\r\n\r\nDescription: \r\n`input {\r\n  udp {\r\n    port => 55144\r\n    type => \"log\"\r\n  }\r\n}\r\noutput {\r\n  stdout { codec => rubydebug }\r\n\r\n  elasticsearch {\r\n        hosts => [\"******\"]\r\n        index => \"alarm\"\r\n        document_type => \"udp\"\r\n        manage_template => true\r\n        template => \"/etc/template/logstash_***.json\"\r\n        template_name => \"***_template\"\r\n        template_overwrite => true\r\n    } \r\n}`\r\n\r\nAfter starting logstash it just keeps giving logs like below:\r\n\r\n> [2019-03-05T06:30:00,457][ERROR][logstash.outputs.elasticsearch] Encountered a retryable error. Will Retry with exponential backoff  {:code=>400, :url=>\"http://elasticsearch.topesc:9200/_bulk\"}\r\n> [2019-03-05T06:30:01,546][ERROR][logstash.outputs.elasticsearch] Encountered a retryable error. Will Retry with exponential backoff  {:code=>400, :url=>\"http://elasticsearch.topesc:9200/_bulk\"}\r\n> \r\n\r\nthe output  _**Codec => ruby ​​debug**_ can output log data, but logstash.outputs.elasticsearch reports an error with\r\n\r\n[ERROR][logstash.outputs.elasticsearch] Encountered a retryable error. Will Retry with exponential backoff {:code=>400, : Url=>\"http://******:9200/_bulk\"}\r\n\r\nElasticsearch does not have any log output,In addition, Elasticsearch is starting and has no error message about this.\r\n\r\nCan you please give some advice on this? Thanks very much!\r\n\r\n\r\n\r\n\r\n","closed_by":{"login":"jasontedor","id":4744941,"node_id":"MDQ6VXNlcjQ3NDQ5NDE=","avatar_url":"https://avatars3.githubusercontent.com/u/4744941?v=4","gravatar_id":"","url":"https://api.github.com/users/jasontedor","html_url":"https://github.com/jasontedor","followers_url":"https://api.github.com/users/jasontedor/followers","following_url":"https://api.github.com/users/jasontedor/following{/other_user}","gists_url":"https://api.github.com/users/jasontedor/gists{/gist_id}","starred_url":"https://api.github.com/users/jasontedor/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jasontedor/subscriptions","organizations_url":"https://api.github.com/users/jasontedor/orgs","repos_url":"https://api.github.com/users/jasontedor/repos","events_url":"https://api.github.com/users/jasontedor/events{/privacy}","received_events_url":"https://api.github.com/users/jasontedor/received_events","type":"User","site_admin":false},"performed_via_github_app":null}